python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -1907,11 +1913,16 @@ def plot_events(self):
             rows.append([
                 f'{(system_time_ns - system_t0_ns) / 1e9:.3f}' if system_time_ns is not None else 'N/A',
                 f'{system_time_ns / 1e9:.3f}' if system_time_ns is not None else 'N/A',
+                'N/A',
```

Comment:
I'd actually just leave this col empty string by default since like 99% of the stuff in the table will not have this column. Makes the reset events stand out more.